src/lib/isLength.js

```diff
@@ -15,5 +15,9 @@ export default function isLength(str, options) {
   const presentationSequences = str.match(/(\uFE0F|\uFE0E)/g) || [];
   const surrogatePairs = str.match(/[\uD800-\uDBFF][\uDC00-\uDFFF]/g) || [];
   const len = str.length - presentationSequences.length - surrogatePairs.length;
-  return len >= min && (typeof max === 'undefined' || len <= max);
+  const isInsideRange = (len >= min && (typeof max === 'undefined' || len <= max));
```

Comment:
question: Are you sure the outer brackets are needed? They were not there in the old code.

Reply:
> question: Are you sure the outer brackets are needed? They were not there in the old code.
No!! Removed them...